docs/README.md

```diff
@@ -0,0 +1,63 @@
+# Bridge-In-Tech Backend Docs
```

Comment:
@mtreacy002 Readme.md for the repo is to get a overview about the project. We can't add docusaurus set up instructions to the main Readme.md as it will be an overhead for someone who doesn't wants to know anything about how documentation website is setup. [This](https://github.com/anitab-org/bridge-in-tech-backend/pull/200/files/3fadd94831327124db764842306d29be57544a67#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938) Readme.md is for someone wanting to contribute to or setup the docusaurus website locally. It is not compulsary but it helps in setting up and deploying the docs website.

Reply:
Also, about the yarn.lock, I see it comitted to github most of the times. yarn.lock files lock the installed dependencies to a specific version and ensure that every install results in the exact same file structure in node_modules across all machines.

Reply:
Ok, it make sense now, @Rahulm2310 . We just need one more reviewer to help with the review.